app/lib/pre_assembly/from_staging_location/file_set.rb

```diff
@@ -4,6 +4,8 @@ module PreAssembly
   module FromStagingLocation
     # Represents a groups of related Files, such as a single master file and the derivatives
     class FileSet
+      VALID_THREE_DIMENSION_EXTENTIONS = ['.obj'].freeze
```

Comment:
EXTENSIONS ?

Reply:
also ... if this is never used, why is it needed?

Reply:
It's used here @ndushay  https://github.com/sul-dlss/pre-assembly/blob/c38ce72946b5a085bdfb656d8a7b79977d1b8374/app/lib/pre_assembly/from_staging_location/file_set.rb#L57